src/images/icons/quiz_1.svg

```diff
@@ -0,0 +1,18 @@
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="512.000000pt" height="512.000000pt" viewBox="0 0 512.000000 512.000000" preserveAspectRatio="xMidYMid meet">
+<metadata fill="#99250a">
+Created by potrace 1.16, written by Peter Selinger 2001-2019
+</metadata>
```

Comment:
Can remove metadata, height, width, preserveAspectRatio etc. Height & Width should be set in the code.